tasmota/include/tasmota_template.h

```diff
@@ -171,7 +172,7 @@ enum UserSelectablePins {
   GPIO_INTERRUPT,
   GPIO_MCP2515_CS,                     // MCP2515 Chip Select
   GPIO_HRG15_TX, GPIO_HRG15_RX,        // Hydreon RG-15 rain sensor serial interface
-  GPIO_VINDRIKTNING_RX,                // IKEA VINDRIKTNING Serial interface
+  GPIO_VINDRIKTNING_RX,                // IKEA VINDRIKTNING Serial Tx
```

Comment:
Looks like a typo, shouldn't this be a "Rx"?

Reply:
Hello Cougar, thank you for your comments, you are right, typo in comment, fixed.